app/models/publication-flow.js

```diff
@@ -22,12 +21,16 @@ export default class PublicationFlow extends Model {
 
   // Belongs To.
   @belongsTo('case') case;
+
   @belongsTo('publication-status', {
     inverse: null,
   }) status;
   @belongsTo('publication-type') type;
 
   // Has many .
+  @hasMany('numac-number', {
+    inverse: null,
```

Comment:
Graag #683 mergen en hier in de plaats `serialize: true` specifiëren. Een numac nummer heeft wel degelijk een inverse. De `inverse: null` wordt hier gebruikt als hack om `serialize: true` te bekomen.

Reply:
Ik zou deze er niet op tegenhouden, als we die ene PR mergen moeten we de relaties bekijken